var/sabath/assets/sabath/datasets/a/autoPhaseNN_aicdi.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
For easier reviews, it may be useful to submit data sets independently of SABATH code.